nima/http2/webserver/src/main/java/io/helidon/nima/http2/webserver/Http2ConnectionProvider.java

```diff
@@ -57,9 +72,22 @@ public Set<String> supportedApplicationProtocols() {
 
     @Override
     public ServerConnection connection(ConnectionContext ctx) {
-        Http2Connection result = new Http2Connection(ctx);
+        Http2Connection result = new Http2Connection(ctx, config);
         result.expectPreface();
 
         return result;
     }
+
+    @Override
+    public List<String> configKeys() {
```

Comment:
This should not be a list

Reply:
Unfortunately I need more than a sngle key in HTTP/1.1 provider. This provider initializes HTTP/2 trough Http2UpgradeProvider and must be able to request/send another provider's config too.